source/js/view.js

```diff
@@ -0,0 +1,220 @@
+let playerInstanceBlock = document.getElementById('player_instance_block');
+let cells = document.getElementsByClassName("cell");
+let availableSymbolList = viewModel.getAvailableSymbolList();
+
+/************ Starting game ************/
+window.onload = function() {
+    let list = viewModel.getPlayerList();
+    rowsInput.value = viewModel.getRows();
+    columnsInput.value = viewModel.getColumns();
+    winLengthInput.value = viewModel.getWinLength();
+
+    for (let i = 0; i < list.length; i++) {
+        getPlayerInstanceTemplate(list[i]);
+    }
+};
+
+function getPlayerInstanceTemplate(player) {
+    let template = document.getElementById('player_instance_template');
+    let playerInstanceList = document.getElementsByClassName('player_instance');
+
+    if ('content' in document.createElement('template')) {
+        playerInstanceBlock.appendChild(template.content.cloneNode(true));
+
+        let playerElem = playerInstanceList[playerInstanceList.length - 1];
+        let optionSymbol = new Option(player.symbol, player.symbol);
+
+        playerElem.setAttribute('id', player.id);
+        playerElem.querySelector('.form-group .player_name').value = player.name;
+        playerElem.querySelector('.form-group .player_symbol').appendChild(optionSymbol);
+        createSymbolList(playerElem.querySelector('.form-group .player_symbol'));
+        createTypeList(player, playerElem.querySelector('.form-group .player_type'));
+    }
+}
+
+function createSymbolList(element) {
+    let selectedSymbol = element.options[element.selectedIndex];
+
+    if (element.options.length > 1) element.options.length = 0;
+    if (selectedSymbol) element.appendChild(selectedSymbol);
+
+    for (let i = 0; i < availableSymbolList.length; i++) {
+        let option = new Option(availableSymbolList[i], availableSymbolList[i]);
+        element.appendChild(option);
+    }
+}
+
+function createTypeList(player, element) {
+    let playerTypeList = viewModel.getPlayerTypeList();
+
+    for (let k in playerTypeList) {
+        if (playerTypeList.hasOwnProperty(k)) {
+            let selectedType = (player.type === k);
+            let option = new Option(k, k, selectedType, selectedType);
+            element.appendChild(option);
+        }
+    }
+}
+
+function startGame() {
+    if (playField.hasChildNodes()) {
+        viewModel.refreshFieldCells();
+        updatePlayerField();
+    }
+    init(rowsInput.value, columnsInput.value);
+}
+
+function reloadGame() {
+    window.location.reload();
+}
+
+function updatePlayerField() {
+    let oldTbody = document.getElementById('fieldTbody');
+    if (oldTbody) {
+        playField.removeChild(oldTbody);
+    }
+}
+
+function init(rows, columns) {
+    viewModel.init();
+    createPlayField(rows, columns);
+    createOccupiedCells(rows, columns);
+    setListenerToCells();
+    changeCurrentStatus();
+}
+
+function createPlayField(rows, columns) {
+    let fieldTbody = playField.appendChild(document.createElement('tbody'));
```

Comment:
If we add `playField.innerHTML = ''` before this line, we can get rid of the `updatePlayerField` function.